### ISSUE TITLE
linking errors in master as of commit b7aac16

### ISSUE THREAD

**Issue by equalsJeffH** [w3c/webauthn#439]
linking errors in master as of commit b7aac16:
```
> githist
*   b7aac16 2017-05-03 | Merge pull request #429 from AngeloKai/angelo-secondAuthnrSelect (HEAD -> master, origin/master, origin/HEAD) [Angelo Liao]
|\  
| * 5606a38 2017-05-03 | added false default to requireResidentKey [AngeloKai]
| * a9a0891 2017-05-03 | Addressed comments by Jeffrey [AngeloKai]
| *   dae8768 2017-05-03 | Fixed comments raised [AngeloKai]
```
```
> curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F force=1 -F output=err
\033[1;96mWARNING:\033[0m The following &lt;var>s were only used once in the document:
  'attachment'
If these are not typos, please add an ignore='' attribute to the &lt;var>.
\033[1;33mLINK ERROR:\033[0m No 'dfn' refs found for 'true'.
&lt;a data-link-type="dfn" data-lt="true">true&lt;/a>
\033[1;33mLINK ERROR:\033[0m No 'dfn' refs found for 'scoped credential' that are marked for export.
&lt;a data-link-type="dfn" data-lt="scoped credential">scoped credential&lt;/a>
\033[1;33mLINK ERROR:\033[0m No 'dfn' refs found for 'rp'.
&lt;a data-link-type="dfn" data-lt="RP">RP&lt;/a>
\033[7;32m ✔ \033[0m Successfully generated, with 3 linking errors
```

**Comment by AngeloKai** [comment]
This will be addressed by https://github.com/w3c/webauthn/pull/448

**Comment by AngeloKai** [comment]
Addressed by #448
